src/main/java/pl/bugdemons/ui/selenide/UiElement.java

```diff
@@ -0,0 +1,10 @@
+package pl.bugdemons.ui.selenide;
+
+import com.codeborne.selenide.SelenideElement;
+
+public interface UiElement extends SelenideElement {
```

Comment:
dałbym bardziej wyrazistą nazwę by podkreślać, że to z tego projektu customowy element, albo że to Selenide na sterydach

`ExtendedSelenideElement`
`BugdemonsElement`

Reply:
done